misc/helm-charts/operator/README.md.gotmpl

```diff
@@ -244,11 +244,11 @@ Or check the `Chart.yaml` file in the `misc/helm-charts/operator` directory:
 apiVersion: v2
 name: materialize-operator
 # ...
-version: v25.1.0-beta.1
-appVersion: v0.125.2  # Use this version for your Materialize instances
+version: v25.1.1
+appVersion: v0.130.3  # Use this version for your Materialize instances
```

Comment:
I wonder since main branch is the next version `v25.2...`, whether these should also be kept in sync with that.  Or whether when we branch later closer to when we start prepping for the v25.2 release candidate, we update then.

Reply:
It's mostly an example, so I'm not sure it matters much.